### DISCUSSION TITLE
Is it possible to disable the hastag-based issue linking?

### DISCUSSION THREAD

**Discussion by pintergreg** [community/community#77944] [Projects and Issues] [answered]
### Select Topic Area

Question

### Feature Area

Issues

### Body

My repo now has two remotes at different hosting services and I would like to selectively disable the hastag based issue linking (#x) from the commit messages and only keep the GH-x based one. Is it possible somehow, maybe with a config file in the .github folder?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by erickdc7** [comment, marked as answer]
As of now, GitHub does not provide a built-in option to selectively disable hashtag-based issue linking within commit messages. The automatic conversion of `#<issue_number>` to a hyperlink referencing an issue is a default behavior on GitHub and cannot be disabled through repository settings or configuration files.

However, you can avoid automatic linking by altering the format of your commit messages. For example, you can include additional text or characters around the issue number to prevent it from being recognized as a link. Instead of writing `#10`, you could write `Issue: #10` or `Fix blah problem (#10)`. This way, GitHub will not automatically create a hyperlink to the issue.

If you want to use a different format for issue references, such as `GH-<issue_number>`, you can continue to do so manually in your commit messages. While this won't disable the hashtag-based linking, it allows you to maintain a consistent reference format that suits your workflow across different hosting services.

For more information on how GitHub handles issue references in commit messages, you can refer to the [GitHub Docs](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests).

**Reply by pintergreg** [reply]
Thank you for the answer. 

I already use GH-10 format, but my history contains #10 formatted issues, which was added to another issue tracker. GitHub as a remote is a new setup. I simply don't want to amend all of the commit messages to prevent linking old commits to the newly added GH issues. That was the idea behind my question.

**Comment by James-E-A** [comment]
Here's a goofy but effective workaround: just inject a comment complaining about the workaround.

```markdown
…in Windows 10 build #<!-- workaround: https://github.com/orgs/community/discussions/77944 -->19041 \(released in May 2020\)…
```

**Reply by svegaca** [reply]
An interesting alternative is to inject `&NoBreak;` like this: `#&NoBreak;10` -> #&NoBreak;10
